examples/pipelinedp4j/MovieMetrics.java

```diff
@@ -21,7 +21,7 @@
  *
  * <p>It is the result of the DP metrics query.
  */
-final class MovieMetrics {
+public class MovieMetrics {
```

Comment:
Changed it to `public final` since kryo complains if not public 
```java.lang.UnsupportedOperationException: com.google.privacy.differentialprivacy.pipelinedp4j.examples.MovieView is not a public class. Only public classes are supported.
	at org.apache.spark.sql.errors.QueryExecutionErrors$.notPublicClassError(QueryExecutionErrors.scala:1721)
	at org.apache.spark.sql.Encoders$.validatePublicClass(Encoders.scala:220)
	at org.apache.spark.sql.Encoders$.genericSerializer(Encoders.scala:230)
	at org.apache.spark.sql.Encoders$.kryo(Encoders.scala:181)
	at org.apache.spark.sql.Encoders$.kryo(Encoders.scala:191)
	at org.apache.spark.sql.Encoders.kryo(Encoders.scala)
	at com.google.privacy.differentialprivacy.pipelinedp4j.examples.SparkExample.readData(SparkExample.java:169)```